### ISSUE TITLE
In Matrix/ Manifest download flows, items should be selectable using keyboard

### ISSUE THREAD

**Issue by adriennes** [DataBiosphere/data-browser#535]
When using the manifest download or matrix download flows, users should be able to select/deselect checkboxes and radio buttons using keyboard only.
